buildtools/mako_build.json

```diff
@@ -74,6 +74,18 @@ _ngeo_folder = '' if ngeo_folder is UNDEFINED else ngeo_folder
       "${_ngeo_folder}externs/mapfish-print-v3.js"
     ],
     "js": [
+% if src is not UNDEFINED:
+% for s in src.split(","):
+    "${s}",
+% endfor
+% endif
+    "${_ngeo_folder}src/**/*.js",
+% if src_set == 'contribs_gmf':
+    "${_ngeo_folder}contribs/gmf/src/**/*.js",
+% endif
+    "node_modules/openlayers/src/**/*.js",
+    "node_modules/openlayers/build/ol.ext/*.js",
+    "node_modules/ol-cesium/src/**/*.js",
```

Comment:
It is a copy of the src block, right? Can we use an intermediate variable to avoid the copy?